xkcd/xkcd.py

```diff
@@ -41,7 +41,7 @@ async def xkcd(self, ctx: commands.Context, comic_number: int = 0):
 
     async def make_comic_embed(self, ctx: commands.Context, data: dict) -> discord.Embed:
         """Generate embed for xkcd comic"""
-        xkcd_embed = discord.Embed(title=f"xkcd Comic: #{data['num']}", colour=await ctx.embed_colour())
+        xkcd_embed = discord.Embed(title=f"[xkcd Comic: #{data['num']}](https://xkcd.com/{data['num']})", colour=await ctx.embed_colour())
```

Comment:
```suggestion
        xkcd_embed = discord.Embed(
            title=f"[xkcd Comic: #{data['num']}](https://xkcd.com/{data['num']})",
            colour=await ctx.embed_colour()
        )
```
To get rid of the line length warning